### DISCUSSION TITLE
Template for 1 gang TX

### DISCUSSION THREAD

**Discussion by ady1972** [SmartHome-yourself/sonoff-tx-ultimate-for-esphome#35] [Q&A]
Hello to everyboby.
I have started using your great project just yesterday and for a first try i bought a 1C ultimate.
Flashing gone fine and integration to Home Assistant too.
Because tx_ultimate_local_us.yaml is compiled for 3 relays, thre touchfiled and also three areas for button (left, middle and right) also these entities are shown in home assistant.
I would like to avoid creations of this entities and i have tried to delete corripondent lines in yaml, but during validation i receive errors.
Is there a template for 1C version ?
Thanks a lot

**Comment by barth11** [comment]
It's in the documentation. Don't change the yaml, change only the parameter.

relay_count (Default: 2)
Indicates whether it is the 1, 2 or 3 relay variant.
The blind configuration only supports the variants with 2 or 3 relays.
Possible Values (Integer 1-3)

**Comment by ady1972** [comment]
Thanks a lot for your answer.
I have already done this. But switch for relay 2,3 and 4 are still available in Home Assistant ESP Device.
![Screenshot 2024-03-26 223217](https://github.com/SmartHome-yourself/sonoff-tx-ultimate-for-esphome/assets/7114188/f2586a1e-7b21-4cd7-bb12-1ad17151f827)

However everything works fine.
I'm still studing to understand if it's possible to have led switched on when realy is on too.  Tomorrow i will receive also 2 and 3 relay version and i will have some other tests.

;)

**Reply by ady1972** [reply]
Thanks for answer i will make some more try :)

**Reply by m4rko1985** [reply]
Ciao scusa ti volevo chiedere se potevamo sentirci in privato ho anche io installato da poco esphome su tx ultimate e volevo chiederti delle cose se possibile

**Reply by ady1972** [reply]
Ciao certamente nessun problema 😄

**Reply by m4rko1985** [reply]
Mi puoi contattare via mail a shuttle2009@libero.it ti lascio il mio cel ci sentiamo su whatsapp o come ti è più comodo

**Comment by barth11** [comment]
I (probably) had similar problem when I was playing with it and flashing it multiple time. Although in my case those were greyed out and in your screenshot, it doesn't appear so.

![image](https://github.com/SmartHome-yourself/sonoff-tx-ultimate-for-esphome/assets/40197317/58b643c6-7352-471d-971d-50294d291f71)

When I had some "old" items, I just went to entities and manually deleted the "garbage" from the other FWs I tested and then I had only the current one. Maybe you can try that but as mentioned, mine were greyed out.

**Comment by ady1972** [comment]
I think i will delete them manually...i like to have entities ....clean ... :)

Thanks a lto and have a nice Easter
